### PR TITLE
Issue 360 add testcase for EnumConverter

### PR DESCRIPTION
**Pull request by jeremysolarz**
This add a test case for EnumConverted as described in issue #360

### CODE REVIEW

File: src/test/java/com/beust/jcommander/JCommanderTest.java [hunk 1]
```diff
@@ -712,6 +713,42 @@ public void variableArityZeroNonBoolean() {
...
+            }
+
+            @Override
+            public Color convert(String value) {
+                return Color.valueOf(value);
+            }
+        }
+
+        class Arg {
+            @Parameter(names = "-colors", converter = ColorEnumConverter.class)
```
Comment (cbeust) [inline src/test/java/com/beust/jcommander/JCommanderTest.java]: I think this whole test should work right out of the box without creating `ColorEnumConverter`, e.g. this should suffice:

            @Parameter(names = "-colors")

So I'm wondering if we really need this test?

Comment (jeremysolarz) [inline src/test/java/com/beust/jcommander/JCommanderTest.java]: Correct it does the idea is to have a working test case with a dummy implementation to see how EnumConverter can be used, and refer to it when an example is needed.

At least in other cases I had the experience that this comes handy sometimes :)

Comment (cbeust) [inline src/test/java/com/beust/jcommander/JCommanderTest.java]: Sounds good. Can you resolve the conflict and push again? I'll merge then.

**Comment by jeremysolarz** [comment]
@cbeust Done :)

File: src/test/java/com/beust/jcommander/JCommanderTest.java (hunk 1 above)
Comment (jeremysolarz) [inline src/test/java/com/beust/jcommander/JCommanderTest.java]: @cbeust I could merge it as well if it is ok for you 😄

Comment (jeremysolarz) [inline src/test/java/com/beust/jcommander/JCommanderTest.java]: @cbeust Thanks!

**Pull request merged**